test/suites/network_zone.sh

```diff
@@ -73,14 +69,18 @@ test_network_zone() {
   done
 
   # Setup DNS peers
-  lxc network zone set lxd.example.net peers.test.address=127.0.0.1
-  lxc network zone set lxdfoo.example.net peers.test.address=127.0.0.1 --project=foo
-  lxc network zone set 2.0.192.in-addr.arpa peers.test.address=127.0.0.1
-  lxc network zone set 0.1.0.1.2.4.2.4.2.4.2.4.2.4.d.f.ip6.arpa peers.test.address=127.0.0.1
+  lxc network zone set lxd.example.net peers.test.address=192.0.2.1
+  lxc network zone set lxdfoo.example.net peers.test.address=192.0.2.1 --project=foo
+  lxc network zone set 2.0.192.in-addr.arpa peers.test.address=192.0.2.1
+  lxc network zone set 0.1.0.1.2.4.2.4.2.4.2.4.2.4.d.f.ip6.arpa peers.test.address=192.0.2.1
+
+  # Enable the DNS listener on the bridge itself
+  lxc config unset core.https_address
+  lxc config set core.dns_address 192.0.2.1:8853
```

Comment:
Since you are using a non-wildcard/non-clashing IP for the DNS address, do we still need to unset/set the `core.https_address` at all?

Reply:
No but should we leave them there for consistency? (I will move the unset back at the top where it belongs)